tuna/miopen/driver/convolution.py

```diff
@@ -168,7 +168,7 @@ def compose_tensors(self, keep_id: bool = False) -> dict:
 
   def get_conv_dict(self) -> dict:
     """Populate c_dict with conv table elems"""
-    c_dict: Dict[str, int] = {}
+    c_dict: Dict[str, Any] = {}
```

Comment:
What is the reason to define it as Any from integer? reason?

Reply:
its got  a string as a value now as well, they are not all integers anymore

Reply:
ok